src/cryptography/hazmat/primitives/serialization/ssh.py

```diff
@@ -572,13 +576,65 @@ def encode_private(
         f_priv.put_sshstr(f_keypair)
 
 
+def load_application(data) -> tuple[memoryview, memoryview]:
+    """
+    U2F application strings
+    """
+    application, data = _get_sshstr(data)
+    if not application.tobytes().startswith(b"ssh:"):
+        raise ValueError(
+            "U2F application string does not start with b'ssh:' "
+            f"({application})"
+        )
+    return application, data
+
+
+class _SSHFormatSKEd25519(_SSHFormatEd25519):
```

Comment:
You shouldn't need to subclass `_SSHFormatEd25519` either, you should just be able to compose with it in `load_public`

Reply:
done